src/test/java/se/sundsvall/supportmanagement/api/NotificationsReadResourceFailureTest.java

```diff
@@ -89,4 +98,61 @@ void getNotificationsWithInvalidOwnerId() {
 		verifyNoInteractions(notificationServiceMock);
 	}
 
+	@Test
+	void getNotificationWithInvalidNamespace() {
+		// Call
+		final var response = webTestClient.get()
+			.uri(builder -> builder.path(PATH_NOTIFICATION).build(INVALID, MUNICIPALITY_ID, NOTIFICATION_ID))
+			.accept(APPLICATION_JSON)
+			.exchange()
+			.expectStatus().isBadRequest()
+			.expectBody(ConstraintViolationProblem.class)
+			.returnResult()
+			.getResponseBody();
+
+		// Verification
+		assertThat(response).isNotNull();
+		verifyNoInteractions(notificationServiceMock);
+	}
+
+
+	@Test
+	void getNotificationWithInvalidMunicipalityId() {
+		// Call
+		final var response = webTestClient.get()
+			.uri(builder -> builder.path(PATH_NOTIFICATION).build(NAMESPACE, INVALID, NOTIFICATION_ID))
+			.accept(APPLICATION_JSON)
+			.exchange()
+			.expectStatus().isBadRequest()
+			.expectBody(ConstraintViolationProblem.class)
+			.returnResult()
+			.getResponseBody();
+
+		// Verification
+		assertThat(response).isNotNull();
+		verifyNoInteractions(notificationServiceMock);
+	}
+
+
+	@Test
+	void getNotificationWithNullNotificationID() {
```

Comment:
Rename this test to something that describes the test better if it shoud be kept as is. Or change the test to verify "invalid" (null?) notificationid (as the testname indicates).